manifest/x86_64/r/ruby_prism.filelist

```diff
@@ -1,35 +1,154 @@
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/sig/prism/dsl.rbs
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/sig/prism/inspect_visitor.rbs
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/sig/prism/lex_compat.rbs
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/sig/prism/mutation_compiler.rbs
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/sig/prism/node.rbs
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/sig/prism/node_ext.rbs
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/sig/prism/pack.rbs
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/sig/prism/parse_result.rbs
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/sig/prism/parse_result/comments.rbs
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/sig/prism/pattern.rbs
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/sig/prism/reflection.rbs
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/sig/prism/relocation.rbs
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/sig/prism/serialize.rbs
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/sig/prism/string_query.rbs
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/sig/prism/visitor.rbs
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/src/diagnostic.c
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/src/encoding.c
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/src/node.c
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/src/options.c
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/src/pack.c
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/src/prettyprint.c
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/src/prism.c
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/src/regexp.c
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/src/serialize.c
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/src/static_literals.c
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/src/token_type.c
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/src/util/pm_buffer.c
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/src/util/pm_char.c
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/src/util/pm_constant_pool.c
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/src/util/pm_integer.c
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/src/util/pm_list.c
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/src/util/pm_memchr.c
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/src/util/pm_newline_list.c
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/src/util/pm_string.c
+/usr/local/lib64/ruby/gems/3.4.0/gems/prism-1.5.0-x86_64-linux/src/util/pm_strncasecmp.c
```

Comment:
it looks like we're just shipping the source code here, rather than the compiled extension